### PR TITLE
Add `rust_random` sample that uses `module_misc_device`.

### PR DESCRIPTION
**Pull request by wedsonaf**
This is based on #219 and #218.

### CODE REVIEW

File: rust/module.rs [hunk 1]
```diff
@@ -683,3 +683,81 @@ pub fn module(ts: TokenStream) -> TokenStream {
...
+    let author = get_byte_string(&mut it, "author");
+    let description = get_byte_string(&mut it, "description");
+    let license = get_byte_string(&mut it, "license");
+    expect_end(&mut it);
+
+    let module = format!("__internal_ModuleFor{}", type_);
+
+    format!(
+        "
+            struct {module} {{
```
Comment (bjorn3) [inline rust/module.rs:730]: Maybe add `#[doc(hidden)]` to hide it from generated documentation?

Edit: should have commented on #219

**Comment by ojeda** [comment]
GitHub should really have a way to see the final changes if a PR is merged without having to go commit by commit.

**Comment by ojeda** [comment]
@alex I assumed you were OK with this given the email; nevertheless please give the sample a second look if you can.

**Comment by alex** [comment]
Yes indeed, still LGTM :-)

**Comment by ojeda** [comment]
Thanks @alex!

**Pull request merged**